### ISSUE TITLE
Biber says it's skipping the name, but skips the entire entry

### ISSUE THREAD

**Issue by moewew** [plk/biber#261]
Via https://tex.stackexchange.com/q/487677/35864

Consider
```latex
\RequirePackage{filecontents}    
\begin{filecontents}{\jobname.bib}
@book{Tegmark1998,
  title   = {Measuring the Galaxy Power Spectrum with Future Redshift Surveys},
  author  = {Emma Sigfridsson},
  editor  = {Tegmark M., Hamilton A.J.S., Strauss M.S., Szalay A.S.},
  journal = {ApJ},
  pages   = {499-555},
  year    = {1998}
}
\end{filecontents}

\documentclass[british]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{babel}
\usepackage[style=authoryear, backend=biber]{biblatex}
\usepackage{hyperref}

\addbibresource{\jobname.bib}

\begin{document}
Lorem \autocite{Tegmark1998}

\printbibliography
\end{document}
```

When compiled with Biber, Biber rightfully complains

```
WARN - Name "Tegmark M., Hamilton A.J.S., Strauss M.S., Szalay A.S." has too many commas: skipping name
```
but then it does not only skip the name. The entire entry is skipped and does not appear in the `.bbl` file.

1. I think it would make sense to promote the warning to an error here, if it has so strong implications.
2. Ideally the message should explain that the entire entry is discarded and not just the name.

Alternatively, it could be considered whether Biber could handle this more gracefully by really just skipping the name and consuming the rest of the entry as expected. I'm not too sure if that is a good idea since the current situation where the entry does not show up at all sends a very clear message that something is wrong. The warning on the other hand might be ignored.

**Comment by plk** [comment]
This is an artefact of only having one name and that name being bad. This leads to a bug where the entire entry is skipped as the name parsing routines return `undef`. However, I agree that we can just throw an error and stop at such names as just dropping names is a bad idea in general. Fixed in 2.13 DEV.

**Comment by moewew** [comment]
Looks good.

**Comment by moewew** [comment]
I only just realised that Biber now completely dies when such an error is occurred and produces no `.bbl` file at all. Previously (>= 2.12) only the problematic entry would be missing from the `.bbl`. Is there any chance to get back the old behaviour in that regard?

```latex
\documentclass[british]{article}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{babel}
\usepackage{csquotes}

\usepackage[style=authoryear, backend=biber]{biblatex}


\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@book{appleby,
  author  = {Humphrey Appleby},
  title   = {On the Importance of the Civil Service},
  date    = {1980},
}
@book{bppleby,
  author  = {Humphrey Bppleby, Gumphrey Shlappleby, Dumfrey Happleby, Drumphrey Tappleby},
  title   = {On the Importance of the Civil Service},
  date    = {1981},
}
@book{cppleby,
  author  = {Humphrey Cppleby},
  title   = {On the Importance of the Civil Service},
  date    = {1982},
}
\end{filecontents}

\addbibresource{\jobname.bib}


\begin{document}
\cite{appleby,bppleby,cppleby}
\printbibliography
\end{document}
```

**Comment by plk** [comment]
You are right, this should work in DEV correctly now.

**Comment by moewew** [comment]
Thanks, this works, but I'm wondering if it might not be better to classify the message in the `.blg` as an error instead of a warning
```
[254] Utils.pm:300> WARN - Name "Humphrey Bppleby, Gumphrey Shlappleby, Dumfrey Happleby, Drumphrey Tappleby" has too many commas, skipping entry 'bppleby'
```

**Comment by plk** [comment]
Can you try DEV again now?

**Comment by moewew** [comment]
Looks good now, thank you.

**Comment by plk** [comment]
Fixed in latest 2.13 release.
